clients/java-logger/shell/src/test/java/com/abcxyz/lumberjack/loggingshell/LoggingControllerTest.java

```diff
@@ -73,19 +78,21 @@ void loggingWithoutTraceIdCausesBadRequestError() throws Exception {
 
   @Test
   void loggingWithTraceIdReturnsOk() throws Exception {
```

Comment:
Nit: we can assert all fields matching our test field in one test instead of multiple tests and each with one assertThat.

Reply:
Yep. I also found other java code in lumberack has such multiple tests style. I will create a separate issue to fix it.

Reply:
https://github.com/abcxyz/lumberjack/issues/392